ignite/contrib/handlers/param_scheduler.py

```diff
@@ -13,6 +13,7 @@
 from torch.optim.optimizer import Optimizer
 
 from ignite.engine import Engine
+from ignite.utils import deprecated
```

Comment:
@fco-dv why not remove all the content of this file, import all from `ignite/handlers/param_scheduler.py` and add a depreciation warning ?

Reply:
thanks @sdesrozis  yes it would be much better